docs/cookbook/drag_and_drop.md

```diff
@@ -0,0 +1,36 @@
+# Creating drag and drop interface
+
+In eZ Platform you are able to create a generic interface for drag and drop interactions that can be reused in many places.
+
+First prepare the HTML code structure that will be placed in a Twig template in the following way:
+
+```html
+<div class="items-container" data-placeholder="HTML template for placeholder"></div>
+```
+
+Next, add options in the same Twig template or in a JavaScript code that comes with the template following the convention:
+
+```javascript
+const draggable = new global.eZ.core.Draggable({
```

Comment:
I think `global` may be confusing a little bit, I would suggest using `window`, so `new window.eZ.core.Draggable(` or adding IIFE like in this guide: https://github.com/ezsystems/developer-documentation/commit/d05663ccf03ee5b7d023a89d518fa74e279f0c0d#diff-1fce18d645fa92f17934470c746a287dR74